bedrock/mozorg/templates/mozorg/home.html

```diff
@@ -112,7 +112,7 @@ <h2 class="panel-title">{{ _('<i>Try the best</i> Firefox yet') }}</h2>
         <h2 class="panel-title">{{ _('<i>Look</i> ahead') }}</h2>
         <div class="panel-inner">
           <div class="panel-content">
-            <a href="{{ url('firefox.os.index') }}">
```

Comment:
Here's another reason not to change the url name. When we deploy the new 'fxos' name won't exist until Apache restarts, but the template may get loaded before that and the page will be broken for a short time.